torch/_inductor/lowering.py

```diff
@@ -4170,41 +4196,90 @@ def avg_pool2d(
     ceil_mode=False,
     count_include_pad=True,
     divisor_override=None,
+):
+    return _avg_poolnd(
+        x,
+        kernel_size,
+        stride,
+        padding,
+        ceil_mode,
+        count_include_pad,
+        divisor_override,
+        dim=2,
+    )
+
+
+@register_lowering(aten.avg_pool3d, type_promotion_kind=None)
```

Comment:
Could do with benchmarking `avg_pool3d` and also check that codegen for `avg_pool2d` hasn't changed.